frontend/src/app/modules/auth/login-page/login-page.component.scss

```diff
@@ -48,7 +54,9 @@
         justify-content: space-between;
 
         .submit-button {
-            cursor: pointer;
+            &:hover {
```

Comment:
I think it should be handled in one place for all the buttons in the app

Reply:
I agree with you, I will fix it